app/public/javascripts/views.js

```diff
@@ -46,6 +57,30 @@ allDays.forEach(function(td){
 ///////////////////////////////////////////////
 ////////////////BOOKING VIEW///////////////////
 ///////////////////////////////////////////////
+
+var CreateBookingView = Backbone.View.extend({
+  render: function(){
+    var date = "This day is " + this.model.get("date");
+    var user = " and the user is " + this.model.get("user");
+    var modal = "<div id='bookingModal' class='modal fade' role='dialog'><div class='modal-dialog'>" +
```

Comment:
Alternatively, if we want to make use of Babel, we could use es6's very awesome template strings feature: https://babeljs.io/docs/learn-es2015/#template-strings